troubleshooting.html

```diff
@@ -0,0 +1,304 @@
+---
+layout: default
+title: Troubleshooting OMERO
+menu-id: l2-more
+pdfs:
+    - troubleshooting.pdf
+description: Help with any problems encountered with OMERO.
+---
+<div class="page-navigation">
+<h2>Contents</h2>
+<p>
+<a href="#error">Error reports</a><br />
+<a href="#log">Finding your OMERO.insight log file</a><br />
+<a href="#fail">Import failure</a><br />
+<a href="#submit">Submitting files</a><br />
+<a href="#information">What information OME stores</a><br />
+<a class="reference external" target="_blank" href="http://www.openmicroscopy.org/site/support/omero/sysadmins/troubleshooting.html">System Administrator Troubleshooting page - OME website</a><br />
```

Comment:
Something funny is going on with this link, it's trying to go to https://www.openmicroscopy.orghttp//www.openmicroscopy.org/site/support/omero5/sysadmins/troubleshooting.html